source/rust_verify/tests/atomic_lib.rs

```diff
@@ -424,17 +424,17 @@ const ATOMIC_I64: &str = code_str! {
 
     let l = at.fetch_or(&mut perm, 2);
     assert(l == 9);
-    assert_bit_vector((9 as u64 | 2 as u64) == 11 as u64);
+    assume((9 as i64 | 2 as i64) == 11 as i64);
```

Comment:
Why are these assumes now?

Reply:
I suggested chanhee change this because these assertions aren't fundamental to the test, and verus apparently lacks the signed int reasoning necessary to make the assertion go through.